core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -373,6 +378,10 @@ public void start(WorkerNetAddress address) throws IOException {
               Configuration.global(), ServerUserState.global()));
     }
 
+    AlluxioConfiguration conf = Configuration.global();
+    String dbDir = conf.getString(PropertyKey.DORA_WORKER_META_STORE_ROCKSDB_DIR);
+    mMetaStore = new RocksDBDoraMetaStore(dbDir + ".worker." + mWorkerId);
```

Comment:
In some tests, alluxio.client.cli.fs.command.DistributedLoadCommandTest, the test framework will start 4 worker process in the same dir.
Before this patch, this is fine.
But now, every worker has its own DoraMeta store dir. We need a way to make each worker use its own dir.

Reply:
can't we configure a different dbDir for each of the worker?

Reply:
Yes, we can. That needs modification to the current test scripts and documents for deploying if we are going to deploy multiple workers on the same node.

Reply:
Will it affect us from reusing the same metastore? e.g. worker id is different when restarting the process

Reply:
You are right.
It's better to have different HOME_DIR to hold these data in RocksDB.
I will investigate a way to do so.
This will be marked as a TODO.